docs/pages/guides/limits.mdx

```diff
@@ -37,5 +37,4 @@ characters once serialized to JSON.
 
 ### Storage
 
-The value of a `LiveObject`, `LiveList`, and `LiveMap` storage item cannot
-exceed 128 kB. This limit can be increased on the Enterprise plan.
+The `LiveObject` and JSON values in storage cannot exceed 128 kB.
```

Comment:
I think we need more info, otherwise people won't fully understand. If @nvie confirms this is correct, I'll give it the ✅.

~~~suggestion
- A `LiveObject` cannot exceed 128 kB when totalling the size of the keys and values. 
- A `LiveMap` can be any size, so long as each individual value does not exceed 128kB.
- A `LiveList` can be any size, so long as each individual value does not exceed 128kB.

Note that when one real-time data structure is nested inside another, it does not count towards the limit. Only the JSON leaves of your data structure count towards the limit. 

For example, given this `LiveObject`:
```ts
const person = new LiveObject({
  name: "Marie",
  traits: new LiveObject({ kind: true }),
  friends: new LiveList(["Albert"]),
  age: 30,
})
```

The data limit for `person` is counted from the following:
```ts
{
  "name": "Marie",
  "age": 30,
}
```

This also applies to values inside `LiveMap` and `LiveList` data structures.
~~~